artix7/primitives/common_slice/CMakeLists.txt

```diff
@@ -5,7 +5,9 @@ add_subdirectory(routing)
 
 add_file_target(FILE common_lut_and_f78mux.model.xml SCANNER_TYPE xml)
 add_file_target(FILE common_lut_and_f78mux.pb_type.xml SCANNER_TYPE xml)
-
+add_file_target(FILE cyinit_fabric.sim.v SCANNER_TYPE verilog)
+v2x(NAME cyinit_fabric SRCS cyinit_fabric.sim.v)
 add_file_target(FILE common_slice.model.xml SCANNER_TYPE xml)
```

Comment:
cyinit_fabric is not needed and should be removed.

Reply:
without `cyinit_fabric.sim.v` the flow fails with:
```
Error 1: 
Type: Blif file
File: symbiflow-arch-defs/build/artix7/tests/counter/counter/artix7-xc7a50t-roi-virt-xc7a50t-test/top.eblif
Line: 12
Message: Failed to find matching architecture model for 'CYINIT_CONSTANTS'

```

Reply:
ignore the above. It was wrong yosys version.

Reply:
`cyinit_fabric` has been removed from this PR